### ISSUE TITLE
[FR] Start a development branch

### ISSUE THREAD

**Issue by andyfarthing** [andrivet/ADVi3pp#49]
Would it be possible to start a development branch going?

There's a few of the more straight-forward things from the FRs that people can implement and make pull requests for, but at the moment we don't really know what's already been sorted privately and so what's worth spending time on. 

For example, I've implemented the BLTouch code from @weidercs' fork, but I've made a few tweaks that speeds up the probing by a fair bit, and which should improve reliability of the readings. I'd make a PR for this, but it looks as though you've already implemented BLTouch privately for testing? My changes are also based on 1.1.6, which is the latest version we have access to on this branch. It looks as though some changes to levelling have been made thru 1.1.8, so they might not even be relevant anymore. 

The main aim is to prevent people from putting in a lot of work, only to find out that it's already been taken care of elsewhere (such as @voretaq7's PR from the other day). It also makes sure that people who *are* working on things are working with the latest sources, so we're not adding things that are no longer relevant or required.

Either way, thanks for the work you've put into this so far - it's definitely one of the more promising firmwares that have been worked on.

**Comment by sniperxnl** [comment]
Do you have a workout for the bltoutch?

**Comment by voretaq7** [comment]
I'd be all for more feature branches in the repo (even if they're unstable) or committing to the head with tags for stable releases if that's your preference - lets us test with the latest code & avoid duplication of effort.

**Comment by andrivet** [comment]
@andyfarthing Yes, I am working on BLTouch support (both SW and HW). Regarding a development branch, I am not sure. I will soon change the way I am publishing new releases and source code. I will re-evaluate a development branch once it is ready (this month probably).

**Comment by andrivet** [comment]
Development branch is called advi3++dev.
